api/v1beta1/disruption_cron_webhook.go

```diff
@@ -196,3 +203,36 @@ func (d *DisruptionCron) validateDisruptionCronSpec() error {
 
 	return nil
 }
+
+func (d *DisruptionCron) validateMinimumFrequency(minFrequency time.Duration) error {
```

Comment:
The Schedule is part of the disruption cron spec. Is it better to do it in the `validateDisruptionCronSpec` method instead? 
The `validateDisruptionCronSpec`  already checks if the `spec.Schedule` is valid.  
WDYT of calling the `validateMinimumFrequency` in the `validateDisruptionCronSpec` method instead?

Reply:
Thanks for the explanations 🙇 
It makes sense for me 👍